aten/src/ATen/ZeroTensorFallback.cpp

```diff
@@ -95,7 +95,9 @@ namespace at {
     m.impl("clone", torch::CppFunction::makeFallthrough());
     m.impl("dot", torch::CppFunction::makeFallthrough());
     m.impl("vdot", torch::CppFunction::makeFallthrough());
-    // The functions in the list below have a specific registeration in native_functions.yaml and
+    m.impl("remainder.Tensor", torch::CppFunction::makeFallthrough()); // XXX: doesn't work
+    m.impl("remainder_jvp", torch::CppFunction::makeFallthrough()); // XXX: doesn't work
+    // The functions in the list below have a specific registration in native_functions.yaml and
```

Comment:
as per my comment below, I don't think these changes are needed

Reply:
yes, i think with these, the code will actually segfault if you pass an undefined tensor as other_t instead of a dual. i'll remove these